README.Rmd

```diff
@@ -103,6 +103,17 @@ babelquarto:
 
 Using `babelquarto::register_main_language()` and `babelquarto::register_further_languages()` will create the boilertemplate for these fields.
 
+## Configure the language link placement
+
+The language link menu can be placed in the *sidebar* or in the *navbar*. To change the placement, use the languagelinks field in the babelquarto YAML key, in `_quarto.yml`:
```

Comment:
```suggestion
The language link menu can be placed in the *sidebar* or in the *navbar*. 
To change the placement, use the languagelinks field in the babelquarto YAML key, in `_quarto.yml`:
```

In Markdown, adding linebreaks at the end of sentences does not change the resulting HTML but it makes reviews and diffs easier.